### PR TITLE
Small fixes for vigiechiro tools

### PR DESCRIPTION
**Pull request by Alanamosse**
In .shed.yml :
 - Correction for the remote repository url.

In BilanEnrichiPF.R  : 
 - Add encoding to accept french special char. 
 - Correct a sep causing wrong display in output hourly table

### CODE REVIEW

File: tools/vigiechiro/BilanEnrichiPF.R [hunk 1]
```diff
@@ -232,6 +232,6 @@ if(substr(IdC2$`nom du fichier`[1],2,2)!="a")
   
   saveWidget(SummHTMLH,"output-hourly.html")
 #  write.csv2(SummPartH,"output-hourly.tabular",row.names=F)
-  write.table(SummPartH,"output-hourly.tabular",row.names=F,sep="\n")
+  write.table(SummPartH,"output-hourly.tabular",row.names=F,sep="\t")
```
Comment (bgruening) [inline tools/vigiechiro/BilanEnrichiPF.R]: Isn't this tested :)

File: tools/vigiechiro/BilanEnrichiPF.R [hunk 2]
```diff
@@ -21,15 +21,15 @@ f2p <- function(x) #get date-time data from recording file names
...
 
 if(substr(IdC2$`nom du fichier`[1],2,2)!="a")
 {
 #  print("Protocole non conforme, ce script doit etre lance uniquement pour un protocole Point Fixe")
   print("Wrong protocol, please only use this tool for a \'Point Fixe\' protocol.")
 }else{
-  refPF=fread(args[2])
-  GroupList=fread(args[3])
+  refPF=fread(args[2],encoding="UTF-8")
+  GroupList=fread(args[3],encoding="Latin-1")
```
Comment (bgruening) [inline tools/vigiechiro/BilanEnrichiPF.R]: Why is this Latin, UTF does not work? Mh this is wired.

Comment (yvanlebras) [reply tools/vigiechiro/BilanEnrichiPF.R]: I agree... I think the problem is the original encoding of the file... We are making test on it

File: tools/vigiechiro/BilanEnrichiPF.R (hunk 1 above)
Comment (Alanamosse) [reply tools/vigiechiro/BilanEnrichiPF.R]: The test compare one out of 3 files, it's pretty cheap you're right :D 
I'll had more

File: tools/vigiechiro/BilanEnrichiPF.R (hunk 2 above)
Comment (Alanamosse) [reply tools/vigiechiro/BilanEnrichiPF.R]: Re-encoding a file resolved this, we should be able to use UTF-8
Thanks @yvanlebras

**Pull request merged**